packages/app/client/src/state/sagas/chatSagas.ts

```
@@ -157,23 +157,24 @@ export class ChatSagas {
     // If an existing factory is found, refresh the token
     const existingFactory: string = yield select(getWebSpeechFactoryForDocumentId, documentId);
     const { GetSpeechToken: command } = SharedConstants.Commands.Emulator;
-    let token;
+
     try {
-      token = yield call(
-        [ChatSagas.commandService, ChatSagas.commandService.remoteCall],
+      const speechAuthenticationToken: Promise<string> = ChatSagas.commandService.remoteCall(
         command,
         endpoint.id,
         !!existingFactory
       );
-    } catch (e) {
-      // No-op - this appId/pass combo is not provisioned to use the speech api
-    }
-    if (token) {
-      const factory = yield call(createCognitiveServicesBingSpeechPonyfillFactory, {
-        authorizationToken: token,
+
+      const factory = yield call(createCognitiveServicesSpeechServicesPonyfillFactory, {
+        authorizationToken: speechAuthenticationToken,
+        region: 'westus2', // Currently, the prod speech service is only deployed to westus2
       });
+
       yield put(webSpeechFactoryUpdated(documentId, factory)); // Provide the new factory to the store
+    } catch (e) {
+      // No-op - this appId/pass combo is not provisioned to use the speech api
```

I think @compulim 's comment about how this `catch` could be hit due to a saga error (`call` or `put`) still stands. However, we currently have no way of distinguishing between the two types of errors.

---

It's a pain but okay to ignore the error for now. 😉